server/api_link.go

```diff
@@ -441,17 +441,29 @@ func (s *ApiServer) LinkGoogle(ctx context.Context, in *api.AccountGoogle) (*emp
 		s.logger.Info("Could not authenticate Google profile.", zap.Error(err))
 		return nil, status.Error(codes.Unauthenticated, "Could not authenticate Google profile.")
 	}
+	
+	displayName := googleProfile.Name
+	if len(displayName) > 255 {
+		// Trim the name in case it is longer than db can store
+		displayName = displayName[0:255]		
```

Comment:
I think we should let displayName to be "" if we can't fully update it. We should also leave a WARN log to tell developers that this has happened.

Reply:
Well! I believe we should truncate the name, or another option could be:
```
	displayName = displayName[0:252] + "..."
```
since (almost) no game will ever display a name that long in full. In most games that I've seen / worked on, we already start truncating the names longer than 20 chars.

Reply:
I side with accuracy, i.e. if we can't update probably it shouldn't be updated. @zyro @novabyte thoughts please.

Reply:
I vote for correctness. If the display name is too long, ignore it and log a warning.

Reply:
Super, will change it to be that way.